backend/src/main/java/gov/cdc/usds/simplereport/db/model/Result.java

```diff
@@ -46,10 +46,8 @@ public class Result extends EternalAuditedEntity {
   @Enumerated(EnumType.STRING)
   private TestResult testResult;
 
-  public Result(
-      TestEvent testEvent, TestOrder testOrder, SupportedDisease disease, TestResult testResult) {
+  public Result(TestEvent testEvent, SupportedDisease disease, TestResult testResult) {
     this.testEvent = testEvent;
-    this.testOrder = testOrder;
     this.disease = disease;
     this.resultLOINC = Translators.convertTestResultToLoinc(testResult);
```

Comment:
non-blocking nit that can go in another PR but I feel like we should rename resultLOINC to resultSNOMED
I found this confusing and we already have enough LOINCs to worry about

Reply:
yeah that a good catch, it will need a db migration to truly change it in the database. I think it belongs in a separate ticket

Reply:
the db col name itself is just `result` which is _technically_ not incorrect but doesn't do much to distinguish it from the other column `test_result` so I could definitely see a ticket for a naming cleanup of these columns and their attributes

Reply:
I included the name of the field here without the db change